### PR TITLE
Dropzone mobile view fix

### PR DESCRIPTION
**Pull request by Flight**
Dropzone fixes for cmsplugin-filer.

### CODE REVIEW

File: filer/static/filer/js/addons/dropzone.init.js [hunk 1]
```diff
@@ -20,8 +20,8 @@
         var mobileClass = 'dropzone-mobile';
         var objectAttachedClass = 'js-object-attached';
         var minWidth = 500;
-        var checkMinWidth = function () {
-            this.toggleClass(mobileClass, this.width() < minWidth);
+        var checkMinWidth = function (element) {
```
Comment (vxsx) [inline filer/static/filer/js/addons/dropzone.init.js]: why not a media query?

Comment (Flight) [inline filer/static/filer/js/addons/dropzone.init.js]: because it can be in popup + it's not counting window width

Comment (vxsx) [inline filer/static/filer/js/addons/dropzone.init.js]: ah, it's more of an element query then, my bad :)

**Comment by vxsx** [comment]
:+1:

**Pull request merged**